allocations/models.py

```diff
@@ -0,0 +1,41 @@
+from django.db import models
+import json
+import logging
+from django.conf import settings
+from datetime import datetime
+
+logger = logging.getLogger('allocations')
+
+class Allocation(models.Model):
```

Comment:
When running this locally and trying to view a project `/user/projects/25142/`, I get the error below,
```
Exception Value: | 'Allocation' object has no attribute 'days_left'
-- | --
/project/projects/views.py in view_project, line 178
```

Reply:
@xandroc You need to use "allocations" branch of pytas. I had a PR for the changes: https://github.com/ChameleonCloud/pytas/pull/1